src/colorizer/viewport/ColorizeCanvas2D.ts

```diff
@@ -36,30 +35,31 @@ import {
   OUT_OF_RANGE_COLOR_DEFAULT,
   OUTLIER_COLOR_DEFAULT,
   OUTLINE_COLOR_DEFAULT,
-} from "./constants";
-import type Dataset from "./Dataset";
-import type { IInnerRenderCanvas } from "./IInnerRenderCanvas";
-import type { RenderCanvasStateParams, RenderOptions } from "./IRenderCanvas";
+} from "src/colorizer/constants";
+import type Dataset from "src/colorizer/Dataset";
 import {
-  type Canvas2DScaleInfo,
-  CanvasType,
   DrawMode,
   FeatureDataType,
   type FrameLoadResult,
   type PixelIdInfo,
   TrackPathColorMode,
-} from "./types";
+} from "src/colorizer/types";
 import {
   computeTrackLinePointsAndIds,
   computeVertexColorsFromIds,
   getGlobalIdFromSegId,
   getLineUpdateFlags,
   hasPropertyChanged,
   normalizePointsTo2dCanvasSpace,
-} from "./utils/data_utils";
-import { convertCanvasOffsetPxToFrameCoords, getFrameSizeInScreenPx } from "./utils/math_utils";
-import { packDataTexture } from "./utils/texture_utils";
-import VectorField from "./VectorField";
+} from "src/colorizer/utils/data_utils";
+import { convertCanvasOffsetPxToFrameCoords, getFrameSizeInScreenPx } from "src/colorizer/utils/math_utils";
+import { packDataTexture } from "src/colorizer/utils/texture_utils";
+import VectorField from "src/colorizer/VectorField";
+import { type Canvas2DScaleInfo, CanvasType } from "src/colorizer/viewport/types";
```

Comment:
A lot of these imports are honestly making me think that `colorizer` should be replaced with:

```txt
lib/
  data/
  viewport/
  state/
```

Where `state` and `viewport` are allowed to import from `data` but not the other way around.